python/pyspark/mllib/classification.pyi

```diff
@@ -90,7 +90,7 @@ class LogisticRegressionWithLBFGS:
 
 class SVMModel(LinearClassificationModel):
     def __init__(self, weights: Vector, intercept: float) -> None: ...
-    @overload
+    @overload  # type: ignore
     def predict(self, x: VectorLike) -> float64: ...
```

Comment:
I can't follow why it complains:

```
python/pyspark/mllib/classification.pyi:93: error: Signature of "predict" incompatible with supertype "LinearClassificationModel"
```

Presumably because of different variable name? we can't change for compatibility reason.